components/raftstore/src/store/worker/read.rs

```diff
@@ -520,16 +524,26 @@ where
             // The local `ReadDelegate` is up to date
             Some(d) if !d.track_ver.any_new() => Some(Arc::clone(d)),
             _ => {
+                if self.get_must_track_version_changed_failpoint() {
+                    if let Some(d) = self.delegates.get(&region_id) {
+                        info!("must_track_version_changed: {}", d.track_ver.any_new());
```

Comment:
What is this log for? While entering this branch already show that the previous `track_ver.any_new()` is true,  call it again immediately usually return false

Reply:
This logic is not changed, but only fix for make clippy pass.